exporter/clickhouselogsexporter/migrations/000006_tag_attributes.up.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE IF NOT EXISTS signoz_logs.tag_attributes ON CLUSTER cluster (
+    timestamp DateTime CODEC(ZSTD(1)), 
+    tagKey LowCardinality(String) CODEC(ZSTD(1)),
+    tagType LowCardinality(String) CODEC(ZSTD(1)),
+    tagDataType LowCardinality(String) CODEC(ZSTD(1)),
+    stringTagValue String CODEC(ZSTD(1)),
+    numberTagValue Nullable(Float64) CODEC(ZSTD(1))
+) ENGINE ReplacingMergeTree
+ORDER BY (tagKey, tagType, tagDataType, stringTagValue, numberTagValue)
+TTL toDateTime(timestamp) + INTERVAL 86400 SECOND DELETE
+SETTINGS ttl_only_drop_parts = 1, allow_nullable_key = 1;
+
+CREATE TABLE IF NOT EXISTS signoz_logs.distributed_tag_attributes ON CLUSTER cluster AS signoz_logs.tag_attributes
+ENGINE = Distributed("cluster", "signoz_logs", tag_attributes, cityHash64(tagKey));
```

Comment:
I think distributed for this task is an overkill.

Reply:
Yeah right it can be overkill, but don't have a strong opinion on this. Do you see any concerns going with distributed?

There might be cases where due to cardinality this table size will be high for logs data but even then a single table should handle it, but keeping it distributed will should make it future proof.

Reply:
Yeah I think we can use distributed.

Reply:
I would suggest pinking a better sharding_key as the`tagKey` may cause uneven distribution since some keys are likely to appear in data more than others, and if they are concentrated on one server, it nullifies the benefit of the distributed.

Reply:
Made the changes, please add in traces as well @makeavish